lib/service/logger.js

```diff
@@ -24,7 +24,7 @@ function _log(level, message) {
     level
   };
 
-  logMessage = JSON.stringify(logMessage) + '\n';
+  logMessage = `${JSON.stringify(logMessage)}\n`;
```

Comment:
I'd rather just have one string because every time you change a string, it creates a new object.